res/values-in/strings.xml

```diff
@@ -20,8 +20,8 @@
     <string name="Avatar_">"Skins:"</string>
     <string name="Name_">"Nama:"</string>
 
-    <string name="Loading___">"Memuat …"</string>
-    <string name="Connecting___">"Menghubungkan …"</string>
+    <string name="Loading___">"unduh …"</string>
+    <string name="Connecting___">"terhubung …"</string>
```

Comment:
Line 23: "U" must be capital letter
Line 24: "T" must be capital letter

Reply:
bro admin  need reason why i change words?

Reply:
It says in the **Tips** and it must be followed to have your changes elligible.